wpilibc/src/main/native/include/frc/controller/LinearPlantInversionFeedforward.h

```diff
@@ -161,14 +136,8 @@ class PlantInversionFeedforward {
   Eigen::Matrix<double, Inputs, 1> Calculate(
       const Eigen::Matrix<double, States, 1>& r,
       const Eigen::Matrix<double, States, 1>& nextR) {
-    if (m_f) {
-      Vector<States> rDot = (nextR - r) / m_dt.to<double>();
-
-      m_uff =
-          m_B.householderQr().solve(rDot - m_f(m_r, Vector<Inputs>::Zero()));
-    } else {
-      m_uff = m_B.householderQr().solve(nextR - (m_A * r));
-    }
+    m_uff = m_B.householderQr().solve(nextR - (m_A * r));
+
```

Comment:
The empty line here doesn't provide any semantic meaning (the reason whitespace like this is normally included). I suggest removing it.